docs/src/pages/components/tabs/HoverTabs.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+export default function CenteredTabs() {
+  const [value, setValue] = React.useState<false | number>(false);
+
+  return (
+    <Box sx={{ width: '100%', bgcolor: 'background.paper' }}>
+      <Tabs value={value} centered onMouseLeave={() => setValue(false)}>
```

Comment:
To reduce the visual distraction of the indicator (moving fewer px)?

```suggestion
      <Tabs value={value} onMouseLeave={() => setValue(false)}>
```